lib/schedulers/ASAPScheduler.js

```diff
@@ -1,19 +1,8 @@
-var asap = require("falcor-asap");
-var empty = {dispose: function() {}};
+var TimeoutScheduler = require("./TimeoutScheduler");
 
-function ASAPScheduler() {}
-
-ASAPScheduler.prototype.schedule = function schedule(action) {
-    asap(action);
-    return empty;
-};
-
-ASAPScheduler.prototype.scheduleWithState = function scheduleWithState(state, action) {
-    var self = this;
-    asap(function() {
-        action(self, state);
-    });
-    return empty;
-};
+// Retained for backwards compatibility
+function ASAPScheduler() {
+    return TimeoutScheduler.bind(this, 1)();
```

Comment:
Can't this just be `TimeoutScheduler.call(this, 1);`?

Reply:
...yes, it can 🤦‍♂